src/realm-settings/ProfilesTab.tsx

```diff
@@ -1,12 +1,184 @@
+        </Link>{" "}
+        {profiles?.globalProfiles?.length! > 0 && (
+          <Label key={`label-${name}`} color="blue">
+            {t("global")}
+          </Label>
+        )}
+      </>
+    );
+  }
+
   return (
-    <PageSection variant="light" padding={{ default: "noPadding" }}>
-      <h1>Profiles Tab</h1>
-    </PageSection>
+    <>
+      <DeleteConfirm />
+      <PageSection>
+        <Flex className="kc-profiles-config-section">
+          <FlexItem>
+            <Title headingLevel="h1" size="md">
+              {t("profilesConfigType")}
+            </Title>
+          </FlexItem>
+          <FlexItem>
+            <Radio
+              isChecked={!show}
+              name="formView"
+              onChange={() => formatChange(false)}
+              label={t("profilesConfigTypes.formView")}
+              id="formView-radioBtn"
+              className="kc-form-radio-btn pf-u-mr-sm pf-u-ml-sm"
+            />
+          </FlexItem>
+          <FlexItem>
+            <Radio
+              isChecked={show}
+              name="jsonEditor"
+              onChange={() => formatChange(true)}
+              label={t("profilesConfigTypes.jsonEditor")}
+              id="jsonEditor-radioBtn"
+              className="kc-editor-radio-btn"
+            />
+          </FlexItem>
+        </Flex>
+      </PageSection>
+      <Divider />
+      {!show ? (
+        <KeycloakDataTable
+          ariaLabelKey="userEventsRegistered"
+          searchPlaceholderKey="realm-settings:clientProfileSearch"
+          isPaginated
+          loader={loader}
+          toolbarItem={
+            <ToolbarItem>
+              <Button
+                id="createProfile"
+                onClick={() => createNewProfile()}
+                data-testid="createProfile"
+              >
+                {t("createClientProfile")}
+              </Button>
+            </ToolbarItem>
+          }
+          actions={[
+            {
+              title: t("common:delete"),
+              onRowClick: () => {
+                toggleDeleteDialog();
+              },
+            },
+          ]}
+          columns={[
+            {
+              name: "name",
+              displayKey: t("clientProfileName"),
+              cellFormatters: [cellFormatter],
+              transforms: [cellWidth(20)],
+            },
+            {
+              name: "description",
+              displayKey: t("clientProfileDescription"),
+            },
+          ]}
+          emptyState={
+            <ListEmptyState
+              message={t("emptyClientProfiles")}
+              instructions={t("emptyClientProfilesInstructions")}
+            />
+          }
+        />
+      ) : (
+        <>
+          <div className="pf-u-mt-md pf-u-ml-lg">
+            <CodeEditor
+              isLineNumbersVisible
+              isLanguageLabelVisible
+              code={JSON.stringify(profiles, null, 2)}
+              language={Language.json}
+              height={"30rem"}
```

Comment:
Strings can be passed directly,
```suggestion
              height="30rem"
```